backend/src/personnel/personnel.service.ts

```diff
@@ -306,7 +307,27 @@ export class PersonnelService {
     availabilityType?: AvailabilityTypeLabel,
     availabilityFromDate?: string,
     availabilityToDate?: string,
+    availableStatus?: string,
+    program?: Program,
   ): Promise<SelectQueryBuilder<T>> {
+    if (availableStatus && availableStatus === 'New') {
+      program === Program.BCWS
```

Comment:
Please make an if else rather than ternary, I think I prefer that

Reply:
Do we need to account for the possibility that ALL could be passed in here? ie. make it if (program === Program.BCWS), else if (program === Program.EMCR)

Reply:
This is coming from the personnel dashboard, which is program specific, so I don't think so